### PR TITLE
mining voucher kits tweak

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl: 
balance: the kits you get from mining voucher have been upgraded try them out to find what is changed
/:cl:

[why]: only the kits with adv scanner were bought and only for that or for crusher memes

webbing kit -> utility kit : added a bunch of trash in it a bruise pack to prevent ppl stealing the medkit , adv scanner to be truly average , a ranged mod so its pretty much  THE default kit/pka based
resonator kit :put the upgraded resonator still meh maybe adding a piece of soap would make it more appealing
minerbot kit :gives an ai upgrade so you can have a friend while you mine, no adv scanner usefull for noobs so they have higher chace of survival 
rescue kit : added a survival pen so you have some actual benefit on the early game, good for medic/hybrid builds
crusher kir: no changes its an offensive build maybe 1 brute patch?

### CODE REVIEW

File: code/modules/mining/machine_vending.dm [hunk 1]
```diff
@@ -169,22 +169,28 @@
...
+			new /obj/item/storage/box/hug/medical(drop_location)
+			new /obj/item/borg/upgrade/modkit/range(drop_location)
+			new /obj/item/t_scanner/adv_mining_scanner(drop_location)
+			new /obj/item/extinguisher/mini(drop_location)
 		if("Resonator Kit")
 			new /obj/item/storage/belt/mining/alt(drop_location)
 			new /obj/item/t_scanner/adv_mining_scanner(drop_location)
 			new /obj/item/extinguisher/mini(drop_location)
-			new /obj/item/resonator(drop_location)
+			new /obj/item/resonator/upgraded(drop_location)
```
Comment (DaxDupont) [inline code/modules/mining/machine_vending.dm]: >upgraded

**Review by DaxDupont — CHANGES_REQUESTED** [review]
No stealth buffs please

File: code/modules/mining/machine_vending.dm (hunk 1 above)
Comment (Tlaltecuhtli) [reply code/modules/mining/machine_vending.dm]: tweak: changed the voucher kits a bit

Comment (DaxDupont) [reply code/modules/mining/machine_vending.dm]: this is a balance change

**Comment by optimumtact** [comment]
fix the changelog entry

**Comment by 81Denton** [comment]
For context:
```
mining_voucher_redeemed across 1212 rounds between 2018-05-01 00:16:23 and 2018-05-28 07:33:35
Crusher Kit 1180
Resonator Kit 830
Survival Capsule and Explorer's Webbing 545
Extraction and Rescue Kit 228
Mining Conscription Kit 65
Minebot Kit 59
```

**Comment by optimumtact** [comment]
Please consider this a friendly warning, your PR's must have a properly descriptive title and body content that explains the purpose of the PR.

Your current title and or body content does not meet requirements, so please update it.

**Comment by Tlaltecuhtli** [comment]
@oranges is the explaination good enough?

**Comment by ExcessiveUseOfCobblestone** [comment]
Stop calling 100% buffs tweaks

Also @Repukan enjoy your field day

**Comment by ExcessiveUseOfCobblestone** [comment]
Also this bypasses a lot of the point costs. After this you'd just be topping off rather than actually progressing.

**Comment by Tlaltecuhtli** [comment]
after early game you just buy medipens and soap as "progress" 
i m trying to balance the less used kits over the adv scanner ones as ppl buy them only for the scanner

**Comment by ExcessiveUseOfCobblestone** [comment]
No....

You buy:
Webbing, Adv Scanner if you didn't choose that one first, Medipens, AI/Minebot, etc.

If adv scanner is so coveted then why not remove the adv scanner from the pool and force people to purchase it?

**Comment by Tlaltecuhtli** [comment]
webbing isnt good most ppl leave it there because you can store all you need in the bag + box
 idk why u getting mad about adding a bruise pack a ranged mod and a beep boop upgrade

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm not getting mad, I just think it's a bad change because a singular item is desired so much that you feel the need to forgo several of the small progression systems to compensate when you could just not have the singular item available roundstart.

**Comment by Repukan** [comment]
PKA upgrade modules are gated behind you actually mining and doing your job so you bite the bullet and spend a sizeable amount of points (1k) to get them, or cooperate with RnD for the upgrades.

Resonators are thrash and no amount of upgrades will make them good.

I think you underestimate how strong having a player controlled, high brute damage dealing, vacuum proof friend is. Even more so at roundstart. It's extremely strong, there's a reason why the board can't be printed in RnD and why you need to invest points to get them.

>Rescue kit description

Are you balancing for fucking league of legends?

**Comment by Tlaltecuhtli** [comment]
mining bots are only good in early game as they wont do much in boss fights and mindless ones are bad bcs you have to micromanage their status so they dont go follow random ores when you dragging them and you dont get adv scanner

rnd doing mining tech happens 95% of the times having 1 range isnt gonna save you from  certain death

**Review by ChangelingRain — COMMENTED** [review]
This is not an acceptable changelog.
Please list all of your changes in the changelog.

**Pull request closed without merging**